emba.sh

```diff
@@ -215,15 +214,44 @@ main()
     fi
   fi
 
+  if [[ $USE_DOCKER -eq 1 ]] ; then
+    if ! command -v docker-compose > /dev/null ; then
+      print_output "[!] No docker-compose found" "no_log"
+      print_output "$(indent "Install docker-compose via apt-get install docker-compose to use emba with docker")" "no_log"
+      exit 1
+    fi
+
+    OPTIND=1
+    ARGS=""
+    while getopts a:A:cdDe:Ef:Fghik:l:m:sz OPT ; do
+      case $OPT in
+        D|f|i|l)
+          ;;
+        *)
+          export ARGS="$ARGS -$OPT"
+          ;;
+      esac
+    done
+
+    print_output "" "no_log"
+    print_output "[!] Emba initializes kali docker container.\\n" "no_log"
+
+    if FIRMWARE="$FIRMWARE_PATH" LOG="$LOG_DIR" docker-compose run emba -c "./emba.sh -l /log/ -f /firmware -i $ARGS" ; then
```

Comment:
could we execute this command alone and evaluate the return value in the if statement?